src/notation/internal/notationinteraction.cpp

```diff
@@ -3089,7 +3089,12 @@ void NotationInteraction::copySelection()
         m_editData.element->editCopy(m_editData);
         Ms::TextEditData* ted = static_cast<Ms::TextEditData*>(m_editData.getData(m_editData.element).get());
         if (!ted->selectedText.isEmpty()) {
-            QGuiApplication::clipboard()->setText(ted->selectedText, QClipboard::Clipboard);
+            #if defined(Q_OS_MAC) || defined(Q_OS_WIN)
```

Comment:
I've seen this piece of code in three places already... I think we should add our wrapper (to the ui module) for QClipboard and use it instead of QApplication::clipboard() throughout the application. The interface for it will look like this:

```
namespace mu::ui {
class IClipboard {
public:
    virtual void setText(const QString& text) = 0;
    virtual QString text() const = 0;
    
    const QMimeData* mimeData() = 0;
    virtual void setMimeData(QMimeData* data) = 0;
}
}
```

And the implementation for the  setText method:

```
void Clipboard::setText(const QString& text) 
{
#if defined(Q_OS_MAC) || defined(Q_OS_WIN)
    QClipboard::Mode mode = QClipboard::Clipboard;
#else
    QClipboard::Mode mode = QClipboard::Selection;
#endif
    QGuiApplication::clipboard()->setText(ted->selectedText, mode);
}
```